services/deployment-agent/docker/healthcheck_curl_host.py

```diff
@@ -0,0 +1,18 @@
+#!/bin/python
+
+import json
+import os
+import sys
+from urllib.request import urlopen
+
+BOOTS_WITH_DEBUGGER = "2"
```

Comment:
the environ created in Dockerfile is called ``SC_BOOT_MODE``

Reply:
thx for that one. fixed.